hoomd/md/integrate.py

```diff
@@ -141,6 +141,7 @@ def _set_rigid(self, new_rigid):
         self._param_dict["rigid"] = new_rigid
 
 
+@hoomd.logging.modify_namespace(("md", "Integrator"))
 class Integrator(_DynamicIntegrator):
     r"""Molecular dynamics time integration.
```

Comment:
What was the logging namespace for this previously? I don't understand why `Integrator`'s logging namespace needs to be manually changed.